src/common/tests/test.txt

```diff
@@ -0,0 +1,3 @@
+char lbl[] = "0123456789";
```

Comment:
This test is failing in Android builds (https://github.com/FirebirdSQL/firebird/actions/runs/8513177447/job/23316639529)

It does not look as a good practice to reference this file based in the source path.

I think it's better to create this file as temporary file in runtime inside the test.

Reply:
Sorry, missed this check.
Is it allowed to use std::filesystem in tests and the android build?
````cpp
namespace fs = std::filesystem;
auto tempFilePath = fs::temp_directory_path() / "test.txt";
```

Reply:
I hope it is. It should be present in ndk 22b and we are using 25b.

Reply:
Should I create a new PR or do you just cherry pick my [postfix](https://github.com/red-soft-ru/firebird/commit/c35ba3f15be8dfe5ed3fa2e19594835df212be1a)?

Reply:
New PR, please.